amulet/level/interfaces/chunk/anvil/anvil_na.py

```diff
@@ -306,6 +312,13 @@ def _unpack_light(
                     ).reshape((16, 16, 16))
         return light_container
 
+    def _decode_islighton(
+        self, chunk: Chunk, data: ChunkDataType, floor_cy: int, height_cy: int
+    ):
+        chunk.misc["isLightOn"] = self.get_layer_obj(
+            data, self.isLightOn, pop_last=True
+        )
+
```

Comment:
This would need to be implemented in the class for the version when this tag was added.
This tag does not exist in this chunk version.